portfolio/src/main/webapp/youtube-style.css

```diff
@@ -0,0 +1,25 @@
+#idea-list {
+  list-style-type: none;
+}
+
+.idea {
+  border: thin solid black;
+  display: flex;
+  margin: 10px;
+  padding: 10px;
+  width: 300px;
+  height: 40px;
+}
+
+.idea span {
+  flex-grow: 1;
+}
+
+.upvote-button {
+  background: url("/images/up_arrow_icon.png");
+  background-size: 100%;
+  border: 0;
+  height:30px;
+  width: 30px;
```

Comment:
super nit: lets make this width first then height like the .idea block above (or switch that one to height then width) just for consistency